### PR TITLE
Improve UI of selected revisions table

### PR DESCRIPTION
**Pull request by kimberlythegeek**
#110

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
 export const treeherderBaseURL = 'https://treeherder.mozilla.org';
 
 export const repoList = ['try', 'autoland', 'mozilla-central'];
+
+export const repoMapping: Dictionary<Repository['name']> = {
+  1: 'mozilla-central',
+  4: 'try',
+  77: 'autoland',
+};
```
Comment (kimberlythegeek) [inline src/common/constants.ts:13]: added this to resolve type error when passing repository to `getTreeherderURL`

File: src/components/Search/SearchResultsListItem.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ function SearchResultsListItem(props: SearchResultsListItemProps) {
   const indexString = index.toString();
 
   const revisionHash = truncateHash(item.revision);
-  const commitMessage: string = getLatestCommitMessage(item);
```
Comment (kimberlythegeek) [inline src/components/Search/SearchResultsListItem.tsx:23]: removed because not necessary

File: src/components/Search/SearchResultsListItem.tsx [hunk 3]
```diff
@@ -45,7 +45,6 @@ function SearchResultsListItem(props: SearchResultsListItemProps) {
           secondary={`${item.author} - ${String(
             new Date(item.push_timestamp * 1000),
           )}`}
-          sx={{ noWrap: 'true' }}
```
Comment (kimberlythegeek) [inline src/components/Search/SearchResultsListItem.tsx:48]: removed, does not do anything

File: src/utils/searchViewHelper.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  )}:${formatNumber(minutes)}`;
+};
+
+const getTreeherderURL = (
+  revision: Revision['revision'],
+  repository: Repository['name'],
+) =>
+  `https://treeherder.mozilla.org/jobs?repo=${repository}&revision=${revision}`;
+
+export { formatDate, getLatestCommitMessage, getTreeherderURL, truncateHash };
```
Comment (kimberlythegeek) [inline src/utils/searchViewHelper.ts:36]: diff of previous helper file to current one: https://www.diffchecker.com/ElVpcJwi

**Review by beatrice-acasandrei — APPROVED** [review]

**Pull request merged**